doc/building.md

```diff
@@ -1,4 +1,57 @@
-Building nyan
+Building the project
 =============
 
-no code yet :)
+Dependencies
+-------------
+
+```
+gcc >= 5.3.0 or clang >= 3.4
+make >= 3.81
+automake >= 1.14
+autoconf >= 2.69
+libtool >= 2.4.2
+bison >= 2.3.0
```

Comment:
we don't have any bison in use yet, i wrote it in the readme before actually implementing the parser, sorry